library/utils.py

```diff
@@ -0,0 +1,107 @@
+import csv
+import json
+import os
+import urllib2
+
+from django.core.files.base import ContentFile
+from django.utils.translation import ugettext as _
+
+OPENLIBRARY_API_URL = 'https://openlibrary.org/api/books?'
+
+
+def to_unicode(text):
```

Comment:
Humm, I'm not sure. It seems we need to specify the encoding, which in our case we don't know, so we need to sort of guess it.